### ISSUE TITLE
mksquashsfs just got reproducible

### ISSUE THREAD

**Issue by eMPee584** [mgorny/squashdelta#4]
Dear Michal,
how does this:
https://github.com/plougher/squashfs-tools/commit/e0d74d07bb350e24efd3100d3798f4f6d893a3d9
and the zstd support included in a "soon" to be released release improve the situation? Because I'm scratching my head how to provide incremental updates to our https://github.com/fsfw-dresden/usb-live-linux in the future, instead of just replacing the whole image...

**Comment by mgorny** [comment]
Yep, that sounds really cool! That said, I don't think squashdelta needs any changes for that, as it doesn't invoke `mksquashfs`. If you build reproducible images, it's just going to be happy about them.

That said, if that option affects compressors somehow, then it would be something for squashdelta to use.
